src/Map/src/Bridge/Google/src/Option/MapTypeControlOptions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Options for the rendering of the map type control.
+ *
+ * @see https://developers.google.com/maps/documentation/javascript/reference/control#MapTypeControlOptions
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final readonly class MapTypeControlOptions
+{
+    /**
+     * @param array<MapTypeId|string> $mapTypeIds
+     */
+    public function __construct(
+        private array $mapTypeIds = [],
+        private ControlPosition $position = ControlPosition::BLOCK_START_INLINE_START,
+        private MapTypeControlStyle $style = MapTypeControlStyle::DEFAULT,
+    ) {
+    }
+
+    public function toArray(): array
+    {
+        return [
+            'mapTypeIds' => array_map(
+                fn (MapTypeId|string $mapTypeId) => $mapTypeId instanceof MapTypeId ? $mapTypeId->value : $mapTypeId,
```

Comment:
So no control here ?  $mapTypeId can be any string ?

Reply:
Updated, added the following code in the constructor:
```php
        $this->mapTypeIds = array_map(
            fn (MapTypeId|string $mapTypeId) => $mapTypeId instanceof MapTypeId ? $mapTypeId : MapTypeId::from($mapTypeId),
            $mapTypeIds
        );
```

Reply:
Hum, in fact, even if a string or `MapTypeId` is allowed (https://developers.google.com/maps/documentation/javascript/reference/3.56/control?hl=en#MapTypeControlOptions.mapTypeIds), since we are on PHP and have native enums, I believe we can simply accept a list of `MapTypeId`, without any strings.